scalameta/semanticdb/shared/src/main/scala/scala/meta/semanticdb/Symbol.scala

```diff
@@ -1,5 +1,4 @@
-package org.langmeta
-package semanticdb
+package scala.meta.semanticdb
```

Comment:
@olafurpg Do we still need `scala.meta.Symbol`?

Reply:
Yes, `Symbol` will need to live with us another cycle, although we might want to deprecate it 🤔